File: ProgramacionII/Personas.java

package persona;


public class Personas {
    
    String nombre;
    
    int cedula;
    
    String telefono;
    
    String correo;
    
    
    public Personas(String nombre,int cedula,String telefono,String correo){
        
       this.nombre = nombre;
       this.cedula = cedula;
       this.telefono = telefono;
       this.correo = correo;
        
    }
    
    public Personas(){
        
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public int getCedula() {
        return cedula;
    }

    public void setCedula(int cedula) {
        this.cedula = cedula;
    }

    public String getTelefono() {
        return telefono;
    }

    public void setTelefono(String telefono) {
        this.telefono = telefono;
    }

    public String getCorreo() {
        return correo;
    }

    public void setCorreo(String correo) {
        this.correo = correo;
    }
    
    
    
    
}

File: ProgramacionII/Personas.py
class Persona:

    def __init__(self, nombre, apellido, cedula, correo, telefono):

            self.nombre = nombre
            self.apellido = apellido
            self.cedula = cedula
            self.correo = correo
            self.telefono = telefono

    
P= Persona("Marbelis","Atencio"," 1063145199","matenciopitalua2005@gmail.com"," 3223474678")

print(" mi nombre es: "+P.nombre,"\n mi apellido es: "+P.apellido,"\n mi cedula es: "+P.cedula,"\n mi correo es: "+P.correo,"\n mi numero de telefono: "+P.telefono)

